static/app/views/dashboardsV2/contextMenu.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {MouseEvent} from 'react';
+import styled from '@emotion/styled';
+import classNames from 'classnames';
+
+import DropdownMenu from 'app/components/dropdownMenu';
+import {IconEllipsis} from 'app/icons';
+
+export const ContextMenu = ({children}) => (
+  <DropdownMenu>
+    {({isOpen, getRootProps, getActorProps, getMenuProps}) => {
+      const topLevelCx = classNames('dropdown', {
+        'anchor-right': true,
+        open: isOpen,
+      });
+
+      return (
+        <MoreOptions
+          {...getRootProps({
+            className: topLevelCx,
+          })}
+        >
+          <DropdownTarget
+            {...getActorProps({
+              onClick: (event: MouseEvent) => {
+                event.stopPropagation();
+                event.preventDefault();
+              },
+            })}
+          >
+            <IconEllipsis data-test-id="context-menu" size="md" />
+          </DropdownTarget>
+          {isOpen && (
+            <ul {...getMenuProps({})} className={classNames('dropdown-menu')}>
```

Comment:
should we use the `'app/components/lists'` here?

Reply:
I don't want to make changes to this component that I didn't really touch except for moving it, maybe I can make a follow up PR